08_project/mohankrishna/cmd/mohankrishna-puppy-server/main_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseArgsOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
```

Comment:
`r` is just testing support. It isn't part of the test's context. It should therefore appear before `// Given`.